.vscode/settings.json

```diff
@@ -1,18 +1,26 @@
 {
     "cSpell.words": [
+        "Arweave",
         "blockhash",
         "callout",
+        "Devnet",
         "hashlist",
         "Hookable",
+        "irsy",
```

Comment:
This send to be wrong

Reply:
this is just the spell checker config. I'll sort that.

Reply:
That’s why I approved nevertheless